src/scripts/components/translations/ConnectToTranslator.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+
+import { TranslationContext } from 'utils/TranslationUtils'
+
+const ConnectToTranslator = Component => {
+  const WrappedComponent = props => (
+    <TranslationContext.Consumer>
+      {translator => <Component {...props} translator={translator} />}
+    </TranslationContext.Consumer>
+  )
+
+  return WrappedComponent
+}
+export default ConnectToTranslator
```

Comment:
Higher order component that connects a component to the translator specified in the translation context